### PR TITLE
chore: fix formatting

### PR DESCRIPTION
**Pull request by MichaelDeBoey**
Maybe it's also a good idea to have a workflow running on each PR to check wether or not the formatting is OK? 🤔

That way, we don't have these errors anymore.

### CODE REVIEW

File: docs/tutorials/jokes.md [hunk 1]
```diff
@@ -1381,7 +1381,7 @@ https://pris.ly/d/getting-started
 
 Now that we've got prisma initialized, we can start modeling our app data. Because this isn't a prisma tutorial, I'll just hand you that and you can read more about the prisma scheme from [their docs](https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference):
 
-```tsx filename=prisma/schema.prisma lines=[13-19]
+```prisma filename=prisma/schema.prisma lines=[13-19]
```
Comment (MichaelDeBoey) [inline docs/tutorials/jokes.md:1384]: As described in https://github.com/remix-run/remix/commit/8b3fc2ecefd621e794038c4eeb00db1006063f64#commitcomment-66400057

https://github.com/remix-run/remix/runs/5136619644?check_suite_focus=true#step:6:1140

CC/ @mcansh

File: docs/tutorials/jokes.md [hunk 2]
```diff
@@ -2157,7 +2157,7 @@ We're going to handroll our own authentication from scratch. Don't worry, I prom
 
 Let's start by showing you our updated `prisma/schema.prisma` file. 💿 Go ahead and update your `prisma/schema.prisma` file to look like this:
 
-```tsx filename=prisma/schema.prisma lines=[13-20,24-25]
+```prisma filename=prisma/schema.prisma lines=[13-20,24-25]
```
Comment (MichaelDeBoey) [inline docs/tutorials/jokes.md:2160]: As described in https://github.com/remix-run/remix/commit/8b3fc2ecefd621e794038c4eeb00db1006063f64#commitcomment-66400057

https://github.com/remix-run/remix/runs/5136619644?check_suite_focus=true#step:6:1141

CC/ @mcansh

File: docs/guides/styling.md [hunk 3]
```diff
@@ -617,7 +617,7 @@ npm add -D sass
 
 2. Add an npm script to your `package.json`'s `script` section' that uses the installed too to generate CSS files.
 
-```js filename="package.json"
+```json filename="package.json"
```
Comment (MichaelDeBoey) [inline docs/guides/styling.md:620]: https://github.com/remix-run/remix/runs/5136619644?check_suite_focus=true#step:6:1132

**Comment by ryanflorence** [comment]
The website doesn't know the prisma language so this would blow up the site, I just pushed a commit that disables the markdown linting until we can get the prisma language in.

I wonder if there's a way to opt-out of linting for a single code block w/o putting an inline comment? (don't want readers thinking they need to disable linting or something)

**Comment by ryanflorence** [comment]
testing out our changing in staging real quick, then we'll deploy the new website and merge this

**Pull request merged**